.config/.env.example

```diff
@@ -1,3 +1,5 @@
 # Database Config
+MONGODB_USER=development
+MONGODB_PASSWORD=
 MONGODB_DATABASE=development
-MONGODB_URI=mongodb://development:development@localhost:27017/${database}
+MONGODB_URI=mongodb://${MONGODB_USER}:${MONGODB_PASSWORD}@localhost:27017/${MONGODB_DATABASE}
```

Comment:
works fine as long as you change **localhost** to **mongodb**.

This is my env file:

`MONGODB_USER=development
MONGODB_PASSWORD=development
MONGODB_DATABASE=development
MONGODB_URI=mongodb://${MONGODB_USER}:${MONGODB_PASSWORD}@mongodb:27017/${MONGODB_DATABASE}`

Reply:
Totally worked, thanks!